universe/package.json

```diff
@@ -2,13 +2,13 @@
   "packagingVersion": "4.0",
   "minDcosReleaseVersion": "1.9",
   "name": "cockroachdb",
-  "version": "2.0.1-2.0.1",
+  "version": "2.0.3-2.0.3",
```

Comment:
Sorry, I know it's not even remotely obvious, but make this "2.0.1-2.0.3". The first number is the version of the framework, the second is the version of cockroach.